Automattic/newspack-theme
# Style 2: Move the all-caps styles to the customizer option.

### All Submissions:


### Changes proposed in this Pull Request:

A continuation of the work started in #303, this PR moves the text-transform styles in Style 2, so they can optionally be turned off in the Customizer, instead of always applied.

See #280.

### How to test the changes in this Pull Request:

1. Apply the PR and run `npm run build`.
2. Navigate to Customizer > Typography and uncheck "Use all-caps for accent text"
3. Check the following locations and pages, to confirm that the text is now not forced to uppercase:

**Main navigation** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63642888-7a800c80-c67b-11e9-9882-4b1f8ecd3eb8.png)

**Main navigation** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642889-7fdd5700-c67b-11e9-9c04-0faef273cb15.png)

**Tertiary Menu** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63642891-879cfb80-c67b-11e9-835d-c7f6f6ea0582.png)

**Tertiary Menu** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642895-8d92dc80-c67b-11e9-81bf-441e5e0ff2c4.png)

**Article Block Section Title** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63642902-a13e4300-c67b-11e9-866d-700dc520e973.png)

**Article Block Section Title** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642899-971c4480-c67b-11e9-9df8-d4f7eb601c2f.png)

**Single Post - category links** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63642907-ac916e80-c67b-11e9-885c-edfb1e87205b.png)

**Single Post - category links** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642926-c92da680-c67b-11e9-8963-08a5828a7534.png)

**Single Post - byline** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63642907-ac916e80-c67b-11e9-885c-edfb1e87205b.png)

**Single Post - byline** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642926-c92da680-c67b-11e9-8963-08a5828a7534.png)

**Single Post - tags** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63642987-f038a800-c67c-11e9-918d-f4649af15720.png)

**Single Post - tags** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642981-e0b95f00-c67c-11e9-9677-725ce3083cf3.png)

**Block - Pullquote citation** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63642991-ff1f5a80-c67c-11e9-9ea5-6c4da2ce8128.png)

**Block - Pullquote citation** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642995-0e9ea380-c67d-11e9-9ebc-115e71ed32f0.png)

**Footer widget title** (box unchecked)

![image](https://user-images.githubusercontent.com/177561/63643002-2aa24500-c67d-11e9-80ed-36b1ea58153c.png)

**Footer widget title** (box checked)

![image](https://user-images.githubusercontent.com/177561/63642996-18280b80-c67d-11e9-8d1a-bfbe38aee1a4.png)

### Other information:
